projects/fw_102_hw/Midnight Sun.code-workspace

```diff
@@ -0,0 +1,11 @@
+{
+	"folders": [
+		{
+			"path": "..\\.."
+		},
+		{
+			"path": "..\\..\\..\\..\\..\\..\\Sheertex\\iot-firmware"
+		}
+	],
+	"settings": {}
+}
```

Comment:
Please don't commit the .code-workspace files VS Code generates; if it's a pain to avoid accidentally committing them then make a PR adding `*.code-workspace` to the .gitignore.